### PR TITLE
Dev layermanager style

### PR DESCRIPTION
**Pull request by alsturm**
- improves and extends layout of layer manager
- prevents transition effects on page load

### CODE REVIEW

File: src/modules/commons/components/icon/Icon.js [hunk 1]
```diff
@@ -39,8 +39,9 @@ export class Icon extends BaElement {
 		this._onClick = () => { };
 		this._disabled = this.getAttribute('disabled') === 'true';
 		this._title = this.getAttribute('title') || '';
-		this._size = this.getAttribute('size') ? parseInt(this.getAttribute('size')) : 25;
+		this._size = this.getAttribute('size') ? parseFloat(this.getAttribute('size')) : 2;
```
Comment (thiloSchlemmer) [inline src/modules/commons/components/icon/Icon.js:42]: if unit of icon size changes to em, please update jsdoc above too (codeline 10)

**Review by thiloSchlemmer — APPROVED** [review]
Nice progress!

File: src/modules/menu/components/contentPanel/contentPanel.css [hunk 2]
```diff
@@ -181,6 +181,7 @@
...
...kZbd2WQeKcr8/eObAE403IOmwiwl4Z5FoBuiJspilHkmdWUWSg3/AHRNY3RzWtdxltLcXlvObj9SWeeFKggiWdm7IolLz4bZ0iytRT5WHsixiRgneBp1PLR2iOaiVljlC2ErJ4GOUMCHZQSgniBX10oRA3UKkYSXOo3S4xjkHBYjSSeYrlG1io5Txsa8dTMbxCHTDNW3Rp0Md0p7i3K0aSY8pSKJ1nWSnyny+YqsEVw3at5jOPHmc5NlttSVcSvgjsq3HnGgmLRAmPbXXE5pDR5m+Y06lcjhKKExFcNWOLq+mh4pfBn+Mphpl2aTYFCyH2GpkGtngjTmFmQj2MsrNo0OoXE/E8ikUiOofJmeYxNpME8fneLx912O3N0K4lDkkKO7iTI8O1oLWJVgIMYkKQsVxdz0ZuT0hxyyHSihCgyyngjwuocNNvOdFiABEFr/AKRDYxP4InARFVhBvVRma0iI7mPa5E9yN9HORdVVfTT+PB3Wj4badVlS6ZTpzUNWNf17COOFhJLyAGrUJojdHfRFVFcg4gmwukjFZJkueaG5vTnZLVrgOcTVpOSMfMa1+iE2tV216KrfmXe+R1NL37571nR9J2u5/wDpOf7ui6f8XUe3tnM0+xv4sux9R0nQs7jp+Lm9vr+0/COq/uPbup5PWa/mdOb/AA04q+m5f4287451HyDpO+w+x9J1v2O3but6jT39Du6j2cX36ld/+I/qXj/xrXldb8c24v8AqZyuh/N9/wDj/Q8/q/Xqf9v7G3jCu0dN2noV/wDZ8vsHeN6dB8l6z2dRr0+3m+vP5nM+xv4tvjHUc3rKv4X3bunYu6dIXrPkfd/7js6npOq/ldJy+V9vfwTt/wAh6/5XQd15XT/Dev7q7l9R1H3+19Fs7bzvZy9n8zdxV/I+6/FtJ/U95+Kdi5HMxj4h3rT+zdq7V1PL7d7eZpz/AHczix7r2v8AQj57H29L036gfAfktd3js/ZPy3e+m6jTd+U6jn/02zjDf0q7X+nfxuq+EfGu2dm7H0AOg7d2n+z6crbu5f8AM14P+DfsJ/vbNfqn+j/x+v8Anprwzlcrl8tOZzuXv27n8r6+3X6fT1+mvGLd/wDjvce/Re1d+7Z03bd7+t53M/M8vqOXs5P2+bv3+mvEnb2PoO4e/Tt/L6LlRNOZ/S/He3acr+o6fm/9/H//2Q==');;
```
Comment (taulinger) [inline src/modules/menu/components/contentPanel/contentPanel.css:184]: For now, we don't want to include any raster images. Would you revert this change, please?

**Review by taulinger — COMMENTED** [review]
Fine!
See the inline comment...

**Review by taulinger — APPROVED** [review]
Let's merge this branch!
